src/markdown-pages/install-with-kurl/cis-compliance.md

```diff
@@ -0,0 +1,68 @@
+---
+path: "/docs/install-with-kurl/cis-compliance"
+date: "2021-12-14"
+weight: 24
+linktitle: "CIS Compliance"
+title: "CIS Compliance"
+isAlpha: false
+---
+The kURL installer can be configured to be Center for Internet Security (CIS) compliant. This is an opt-in feature that is configured in the kURL specification by setting the `kurl.cisCompliance` field to `true`. For information about known limitations, see [Known Limitations](#known-limitations). For more information about CIS security compliance for Kubernetes, see the [CIS benchmark information](https://www.cisecurity.org/benchmark/kubernetes).
+
+The following settings are changed when `cisCompliance` is set to `true`:
+
+**Primary node configuration:**
+
+The admin.conf file ownership is set to `root:root`.
+
+**API server configuration:**
+
+* `--kubelet-certificate-authority` is set as appropriate.
```

Comment:
Is there more detail we can add to "as appropriate", or is this assumed knowledge? I suppose it is set to the certificate authority that the vendor uses for their API server configuration? (or something like that?)

Reply:
I checked with Jalaja, and this is assumed knowledge.